### ISSUE TITLE
Orange prompts for version update but never actually updates

### ISSUE THREAD

**Issue by Unis0926** [biolab/orange3#5589]
**What's wrong?**

Orange prompts me to update its version when running programs from 3.26 to 3.29. I follow the prompt to update and it appears to update in my Anaconda Navigator. After its complete, I restart Orange to no change. 

**How can we reproduce the problem?** 

Honestly I don't know. I just open the software from Anaconda and then select my working program and the prompt appears at the bottom right corner. 

**What's your environment?**

- Operating system: Windows 10
- Orange version: 3.26.0
- How you installed Orange: Anaconda Powershell

**Comment by markotoplak** [comment]
Could you tell us what happens in the updating process? Does it open a web page where you download the package and then run the installer or does something else happen?

**Comment by Unis0926** [comment]
No. It appears to update in Anaconda Navigator and runs through those steps until it says that I can then launch Orange again.

**Comment by markotoplak** [comment]
Could you post a screenshot of the prompt and of the update procedure? Thanks!
